### ISSUE TITLE
Please let me set the version in a config file somewhere

### ISSUE THREAD

**Issue by chrisnicola** [AppGyver/steroids#885]
Updating the versions through the web interface is tedious and doesn't really make any sense. Version numbers should be checked in with the code and ideally set in one file, once for all builds of that commit/deploy.

Setting it individually for each Android and iOS build type makes this even worse. There needs to be a better way to do this.

**Comment by harsaharsa** [comment]
Yup, I agree. Especially annoying when e.g. bumping iTunes Connect versions where each build needs to be incremented. It's on the feature set for the Build Service API epic (no ETA, but it's high on the backlog).

**Comment by chrisnicola** [comment]
Also would be good if other options could be checked in. Icon/Splash screen assets, plugin config, etc.

**Comment by emc2spider** [comment]
+1

**Comment by roberto-robles** [comment]
+1

**Comment by garrettlangley** [comment]
:+1:

**Comment by benjamincharity** [comment]
:+1:
